src/hotspot/cpu/x86/stubGenerator_x86_64.cpp

```diff
@@ -6966,6 +6966,331 @@ address generate_avx_ghash_processBlocks() {
+        StubRoutines::_vector_tanh_float64    = (address)os::dll_lookup(libsvml, "__svml_tanhf4_ha_ex");
+        StubRoutines::_vector_tanh_float128   = (address)os::dll_lookup(libsvml, "__svml_tanhf4_ha_ex");
+        StubRoutines::_vector_tanh_double64   = (address)os::dll_lookup(libsvml, "__svml_tanh1_ha_ex");
+        StubRoutines::_vector_tanh_double128  = (address)os::dll_lookup(libsvml, "__svml_tanh2_ha_ex");
+        StubRoutines::_vector_log_float64     = (address)os::dll_lookup(libsvml, "__svml_logf4_ha_ex");
+        StubRoutines::_vector_log_float128    = (address)os::dll_lookup(libsvml, "__svml_logf4_ha_ex");
+        StubRoutines::_vector_log_double64    = (address)os::dll_lookup(libsvml, "__svml_log1_ha_ex");
+        StubRoutines::_vector_log_double128   = (address)os::dll_lookup(libsvml, "__svml_log2_ha_ex");
+        StubRoutines::_vector_log10_float64   = (address)os::dll_lookup(libsvml, "__svml_log10f4_ha_ex");
+        StubRoutines::_vector_log10_float128  = (address)os::dll_lookup(libsvml, "__svml_log10f4_ha_ex");
+        StubRoutines::_vector_log10_double64  = (address)os::dll_lookup(libsvml, "__svml_log101_ha_ex");
+        StubRoutines::_vector_log10_double128 = (address)os::dll_lookup(libsvml, "__svml_log102_ha_ex");
+        StubRoutines::_vector_log1p_float64   = (address)os::dll_lookup(libsvml, "__svml_log1pf4_ha_ex");
+        StubRoutines::_vector_log1p_float128  = (address)os::dll_lookup(libsvml, "__svml_log1pf4_ha_ex");
+        StubRoutines::_vector_log1p_double64  = (address)os::dll_lookup(libsvml, "__svml_log1p1_ha_ex");
+        StubRoutines::_vector_log1p_double128 = (address)os::dll_lookup(libsvml, "__svml_log1p2_ha_ex");
+        StubRoutines::_vector_atan2_float64   = (address)os::dll_lookup(libsvml, "__svml_atan2f4_ha_ex");
+        StubRoutines::_vector_atan2_float128  = (address)os::dll_lookup(libsvml, "__svml_atan2f4_ha_ex");
+        StubRoutines::_vector_atan2_double64  = (address)os::dll_lookup(libsvml, "__svml_atan21_ha_ex");
+        StubRoutines::_vector_atan2_double128 = (address)os::dll_lookup(libsvml, "__svml_atan22_ha_ex");
+        StubRoutines::_vector_hypot_float64   = (address)os::dll_lookup(libsvml, "__svml_hypotf4_ha_ex");
+        StubRoutines::_vector_hypot_float128  = (address)os::dll_lookup(libsvml, "__svml_hypotf4_ha_ex");
+        StubRoutines::_vector_hypot_double64  = (address)os::dll_lookup(libsvml, "__svml_hypot1_ha_ex");
+        StubRoutines::_vector_hypot_double128 = (address)os::dll_lookup(libsvml, "__svml_hypot2_ha_ex");
+        StubRoutines::_vector_cbrt_float64    = (address)os::dll_lookup(libsvml, "__svml_cbrtf4_ha_ex");
+        StubRoutines::_vector_cbrt_float128   = (address)os::dll_lookup(libsvml, "__svml_cbrtf4_ha_ex");
+        StubRoutines::_vector_cbrt_double64   = (address)os::dll_lookup(libsvml, "__svml_cbrt1_ha_ex");
+        StubRoutines::_vector_cbrt_double128  = (address)os::dll_lookup(libsvml, "__svml_cbrt2_ha_ex");
```

Comment:
May be use macros with **comments**?  In this code is very easy make `copy-paste` errors. And It is very not clear for me what all these numbers and letters mean in names. I can guess but it should be comments.